things/service_test.go

```diff
@@ -64,6 +75,20 @@ func TestCreateThings(t *testing.T) {
 			token:  wrongValue,
 			err:    things.ErrUnauthorizedAccess,
 		},
+		{
+			desc:   "create new things with external UUID",
+			things: []things.Thing{{ID: "fe6b4e92-cc98-425e-b0aa-000000000001", Name: "a", Metadata: meta},
```

Comment:
Create a `var` for this lists. For example `thingsExtID`

Reply:
User var.